docs/sources/panels-visualizations/visualizations/traces/index.md

```diff
@@ -13,16 +13,99 @@ weight: 850
 
 # Traces panel
 
-{{% admonition type="note" %}}
-This panel is currently in beta. Expect changes in future releases.
-{{% /admonition %}}
-
-_Traces_ are a visualization that enables you to track and log a request as it traverses the services in your infrastructure.
+Traces let you follow a request as it traverses the services in your infrastructure.
+The Traces panel visualizes traces data into a diagram that allows you to easily interpret it.
 
 For more information about traces and how to use them, refer to the following documentation:
 
-- [What are traces](/docs/grafana-cloud/traces)
 - [Tracing in Explore]({{< relref "../../../explore/trace-integration/" >}})
+- [Tempo data source]({{< relref "../../../datasources/tempo/" >}})
 - [Getting started with Tempo](/docs/tempo/latest/getting-started)
 
 {{< figure src="/static/img/docs/explore/trace-view-9-4.png" class="docs-image--no-shadow" max-width= "900px" caption="Screenshot of the trace view" >}}
+
+## Add a panel with tracing visualizations
+
+Once you have tracing data available in your Grafana stack, you can add tracing panels to your Grafana dashboards.
+
+Using a dashboard variable, `traceID`, lets you create a query to show specific traces for a given trace ID.
+For more information about dashboard variables, refer to the [Variables documentation]({{< relref "../../../dashboards/variables" >}}).
+
+### Before you begin
+
+To use this procedure, you need:
+
+- A Grafana instance
+- A Tempo data source connected to your Grafana instance (see [documentation](/docs/grafana/datasources/tempo/))
+
+### Add the Traces panel query
+
+To view and analyze traces data in a dashboard, you need to add the tracing panel to your dashboard and define a query using the panel editor.
+The query determines the data that is displayed in the panel.
+For more information on the panel editor, refer to the [Panel editor documentation]({{< relref "../../panel-editor-overview" >}}).
+
+This procedure uses dashboard variables and templates to allow us to enter trace IDs which can then be visualized. We’re going to use a variable called `traceId` and add it as a template query.
+
+1. From your Grafana instance, create a new dashboard or go to an existing dashboard where you would like to add tracing panels.
+1. Select **Add visualization** from a new dashboard or select **Add Panel** on an existing dashboard.
+1. Search for and select the appropriate tracing data source.
+1. In the panel search pane on the right, select the **Visualizations** tab, search for, and select **Traces**.
```

Comment:
I don't think we call this the search pane and the Viz picker is above the search bar, so I'm thinking just go with a more directional reference. 
```suggestion
1. In the top-right of the panel editor, select the **Visualizations** tab, search for, and select **Traces**.
```